ansible/configs/satellite-v64-prod/env_vars.yml

```diff
@@ -183,6 +186,10 @@ capsule_instances:
     count: "{{capsule_instance_count}}"
     security_groups:
       - CapsuleSG
+<<<<<<< HEAD
+      - DefaultSG
```

Comment:
Adding this re-breaks Openstack deployments

Reply:
fixed